### ISSUE TITLE
part-CNN training problem?

### ISSUE THREAD

**Issue by whyou5945** [Jianlong-Fu/Multi-Attention-CNN#3]
i'm wondering whether anyone retrain the model and achieve the accuracy as reported in the paper? now i'm stuck on the part-cnn training process. problems are listed below:
1. the data to the part-cnn, how to get? by running the attention model on the training dataset and crop?
2. if we get the data, how to train the six parts? finetune every part individually then finetune together? or we just finetune all together?
thanks for answering.
hope to get answer from the author @Jianlong-Fu , looking forward.

**Comment by Heliang-Zheng** [comment]
1. The data for part-cnn are obtained by running the attention model on the training dataset and crop.
2. Fine tune every part individually, then extract the parts' 512-dim features and concatenate them together to train a fc layer for the final classification. Tips: a large drop_ratio for the drop out layer would be helpful.

**Comment by Heliang-Zheng** [comment]
@whyou5945 
1. The data for part-cnn are obtained by running the attention model on the training dataset and crop.
2. Fine tune every part individually, then extract the parts' 512-dim features and concatenate them together to train a fc layer for the final classification. Tips: a large drop_ratio for the drop out layer would be helpful.

**Comment by kli017** [comment]
@whyou5945 Hello, Do you retrained the part-cnn? I was trying to retrain the model on CUB but it seems do not converge.
